src/components/TextInput/TextInput.module.css

```diff
@@ -0,0 +1,17 @@
+.password_container{
+    position: relative;
+}
+
+.password_icon{
+    top:-3px;
+    font-size: 1.6rem;
+    position: absolute;
+    right: 5px;
+    cursor: pointer;
+}
+
+
+.input{
+    width: 95%;
+    margin-top: var(--sp-0-5);
```

Comment:
use better naming conventions.

Reply:
Changed spacing variables to --rem-unitplace-decimalplace format, like --rem-1 for 1rem spacing.